code/game/machinery/computer/card.dm

```diff
@@ -86,7 +86,7 @@
 
 	var/data[0]
 	data["src"] = "\ref[src]"
-	data["station_name"] = station_name()
+	data["server_name"] = CONFIG_GET(string/server_name)
```

Comment:
This one should be ship name, but will require more modifications if you alter the data index.